src/backend/parser/parse_expr.c

```diff
@@ -931,6 +931,25 @@ rewrite_scope_identity_call(ParseState *pstate, Node **lexpr, Node **rexpr)
 	if (!(*lexpr) || !(*rexpr))
 		return;
 
+	func_expr = (FuncExpr *) (*rexpr);
+	if (IsA(*rexpr, FuncExpr) &&
+		strstr(get_func_name(func_expr->funcid), "like_escape") != NULL)
+	{
+		if((func_expr->args)->length==2)
+		{
+			Node *node = lsecond(func_expr->args);
+			if (IsA(node,Const) && ((Const *)(node))->constisnull)
+			{
+				/*
+				 * This condition handles like escape null query,
+				 * only when using TSQL.
+				 */
+				*rexpr = (Node *)((func_expr)->args->elements[0]).ptr_value;
```

Comment:
this is not right way to access list element directly. Use appropriate API form list.h